lib/python/test_file_classifier.py

```diff
@@ -263,6 +263,17 @@ def test_velocity(self):
         self.assertEqual(dest_dir, 'IMOS/ANMN/NRS/NRSROT/Velocity')
         self.assertEqual(dest_filename, filename)
 
+        filename = 'IMOS_ANMN-NRS_AETVZ_20150703T053000Z_NRSROT-ADCP_FV00_NRSROT-ADCP-1507-Workhorse-ADCP-43_END-20151023T034500Z_C-20151117T074309Z.nc'
+        testfile = os.path.join(self.tempdir, filename)
+        make_test_file(testfile, {'site_code':'NRSROT'},
+                       UCUR_MAG={},
+                       VCUR_MAG={},
+                       CSPD={}
+        )
```

Comment:
FYI, I now use a vim plugin in my vimrc (https://github.com/tell-k/vim-autopep8)
```
Plugin 'tell-k/vim-autopep8' 
let g:autopep8_select="W293,E261,E302,F405,E225,E203,E231,E305,E502,E124,E303,E266" 
```
with a list of PEP8 I want to fix automatically. In vim I press F8, and those get fixed automatically